packages/@okta/vuepress-site/docs/reference/api/roles/index.md

```diff
@@ -3099,13 +3103,22 @@ The following `_links` are returned:
 
 A resource has an ID and a link that points to the resource. Supported resources are:
 
-* Apps
-* Groups
-* All Users within a Group
-* All Users within the org
-* All Groups within the org
-* All Apps within the org
-* All Apps of the same type
+#### Supported REST URL resources
```

Comment:
```suggestion
#### Supported resources
```
Is it ok if we remove "REST URL" in the title if the table contains both ORN and REST URL?